tox.ini

```diff
@@ -1,3 +1,18 @@
 # SPDX-License-Identifier: MIT
 [lsr_config]
 lsr_enable = true
+
```

Comment:
the remainder is probably copied/pasted from the storage role - we don't need this for the snapshot role

Reply:
Ok.